docker-compose.yml

```diff
@@ -54,6 +54,37 @@ services:
       - "/var/run/docker.sock:/var/run/docker.sock"
     container_name: logspout
     command: syslog+tls://${ELK_ENDPOINT}
+  
+  cadvisor:
+    restart: on-failure
+    image: gcr.io/google-containers/cadvisor:latest
+    ports:
+      - 8080:8080
```

Comment:
use `expose` instead of `ports`. e.g. `expose 8080`. We just want to expose this service in internal `monitor_net` instead of to outside world.